Index.html

```diff
@@ -1,269 +1,176 @@
-        </div>
-        <!-- only add the toolbar__open__rowx using javascript -->
-        <nav class="drop-down menu__button__align mobile__header">
-            <button id="hamburger">
-                <i class="fa fa-bars closed__menu"></i>
-                <img class="opened__menu cross" src="./Contents/Resources/Icons/cross.svg" alt="">
-            </button>
-        </nav>
-
-        <img class="mobile__photos mobile__invisible" src="./Contents/Resources/Icons/nav_dots.svg" alt="">
 
-        <ul class="mobile__invisible desktop__header ">
-            <li class="header__navication"><a class="nav__desktoplink" href="#headline">Hello</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#works">Works</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#about">About</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#contact">Contact</a></li>
+<body>
+  <header id="header">
+    <nav id="header_nav">
+      <i class="fa-solid fa-bars" id="hamburger-menu"></i>
+      <ul id="desktop-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+      <i class="fa-solid fa-xmark" id="closeMenuBtn"></i>
+      <ul id="mobile-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <section id="intro">
+    <div>
+      <h1>Hi, I'm <b>Mohammad Mehdi Pazouki</b>, Happy to see you</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="dark small">Start collaboration</button>
+      <div id="logo"></div>
+    </div>
+  </section>
+  <section id="Projects">
+    <h1>Projects</h1>
+  </section>
+  <section id="about">
+    <div id="about-info">
+      <h1>About</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+    </div>
+    <button type="button" class="dark small">Get my CV &#10515;</button>
+    <ul id="skills-and-frameworks">
+      <li>Languages
+        <ul id="Languages">
+          <li>JavaScript</li>
+          <li>Python</li>
+          <li>HTML</li>
+          <li>CSS</li>
         </ul>
```

Comment:
- [ ] There are some issues in this about section. kindly consider fixing them following figma guidelines.
   -  There is not enough padding at left and right side of about section. kindly consider increasing the padding on higher displays, for example greater than 1200px.
   - The About heading font is different and smaller in your design.
   - The p tag font-size is bigger in your design.
   - And the padding of `Get my CV` button is lower in your design so consider increasing it a bit.
your design
![n](https://user-images.githubusercontent.com/96683843/236271027-87460595-4550-4d88-a065-f6425657c43b.jpg)

figma design
![c](https://user-images.githubusercontent.com/96683843/236271046-f989c706-6e90-45d8-80a9-191340c53918.jpg)